src/app/wizards/dataModel/data-model-main/data-model-main.component.ts

```diff
@@ -50,6 +59,7 @@ export class DataModelMainComponent implements OnInit {
 
   ngOnInit() {
     this.title.setTitle('New Data Model');
+    this.model = { metadata :[], classifiers : [], label:undefined, description:undefined, author:undefined, organisation: undefined, dataModelType:undefined,dialect:undefined, selectedDataTypeProvider:undefined};
```

Comment:
If something can optionally be `undefined` I don't think you have to explicitly set this on object creation. Makes no difference though.